ozaria/site/views/play/level/tome/SpellView.coffee

```diff
@@ -564,6 +565,10 @@ module.exports = class SpellView extends CocoView
     @createToolbarView()
     @updateHTML create: true if @options.level.isType('web-dev')
 
+  onRecompile: ->
+    @recompile
```

Comment:
Left by mistake?

Reply:
This is required because the updating of the spell has been disabled as part of this PR. If we only send the event, it won't update everything it needs to. To recompile and send the event was what covered all the use cases.